Lib/typing.py

```diff
@@ -1394,10 +1394,24 @@ def _get_protocol_attrs(cls):
                 attrs.add(attr)
     return attrs
 
+_classvar_prefixes = ("typing.ClassVar[", "t.ClassVar[", "ClassVar[")
```

Comment:
What about raw `ClassVar` annotation with generic argument?

Reply:
Interesting, I didn't know that was allowed, but mypy --strict doesn't seem to complain so I guess it is!